lib/src/test/java/vmath/Mat2Test.java

```diff
@@ -22,6 +22,42 @@ void givenIndices_whenGetCalled_thenReturnValueMatches() {
         //TODO
     }
 
+    @Test
+    void givenArgs_whenMat2Created_thenMatrixHasGivenValues() {
+        float[] validValues = {
+                2.0f, 3.0f,
+                4.0f, 5.0f
+        };
+
+        Mat2 mat2 = new Mat2(validValues);
+
+        for (int i = 0; i < 2; i++) {
+            for (int j = 0; j < 2; j++) {
+                assertEquals(validValues[i * 2 + j], mat2.get(i, j));
+            }
+        }
+    }
+
+    @Test
+    void givenWrongSize_whenMat2Created_thenThrowsException() {
+        float[] invalidSize = {
+                2.0f, 3.0f,
+                4.0f
```

Comment:
Indentation

Reply:
Rename the method to `givenArrayOfInvalidLength_whenMat2Created_thenThrowsException`

Reply:
Sorry, I did not see the top messages. Will fix them in a moment